doc/examples/probabilistic_fiber_tracking.py

```diff
@@ -65,8 +66,8 @@
 pmf = fod.clip(min=0)
 prob_dg = ProbabilisticDirectionGetter.from_pmf(pmf, max_angle=30.,
                                                 sphere=small_sphere)
-streamlines = LocalTracking(prob_dg, classifier, seeds, affine, step_size=.5)
-save_trk("probabilistic_small_sphere.trk", streamlines, affine, labels.shape)
+streamlines_generator = LocalTracking(prob_dg, classifier, seeds, affine, step_size=.5)
+save_trk("probabilistic_small_sphere.trk", Streamlines(streamlines_generator), affine, labels.shape)
```

Comment:
What happens if `save_trk` gets the generator? Maybe we should allow that and generate inside of `save_trk`?

Reply:
it works, I update this. Thanks !